content/en/workshop/index.adoc

```diff
@@ -6,69 +6,105 @@ images: []
 
 == Important Dates
 
 [options="header"]
 |===
 | Event | Date
-| Workshop Paper Submission Deadline | TBA
-| Notification of Acceptance | TBA
-| Camera-ready Papers Due | TBA
-| Workshop Dates | TBA
-| Main Conference Dates | TBA
+| First call for papers | February 7, 2024 (Wednesday)
+| Secodn call for papers | March 4, 2024 (Monday)
+| Workshop Paper Submission Deadline | May 31, 2024 (Friday)
+| Notification of Acceptance | June 17, 2024 (Monday)
+| Camera-ready Papers Due | July 1, 2024 (Monday)
+| Workshop Dates | August 15-16, 2024 (Thursday-Friday)
 |===
 
 Note: All deadlines are 11:59 pm UTC -12h (“anywhere on Earth”).
 
 == Contact Information
 
-* Email: ....
-* Submission Portal: ...
+* Email: sigturk.secretary@gmail.com (TODO: change)
+* Submission Portal: https://openreview.net/group?id=aclweb.org/ACL/2024/Workshop/SIGTURK
 * Official Website: https://sigturk.com/workshop
 
 == Organizing Committee
 
-* ...
+* Duygu Ataman
+* Deniz Zeyrek Bozşahin
+* Mehmet Oguz Derin
+* Sardana Ivanova
+* Jonne Sälevä
 
 == Invited Speakers
 
-* ...
+* Kemal Oflazer, CMU
 
 == Program Committee
 
-* ...
+* Askar Aituov, Google
+* Necva Bölücü, CSIRO
+* Çağrı Çöltekin, University of Tübingen
+* Ebru Ersöyleyen, Middle East Technical University
+* Orhan Fırat, Google Deepmind
+* Omer Goldman, Bar-Ilan University
+* Mammad Hajili, Microsoft
+* Rasul Karimov, Sharechat
+* Bekhzod Khoshimov, UW-Madison
+* Abdullatif Köksal, LMU Munich
+* Murathan Kurfalı, Stockholm University
+* Constantine Lignos, Brandeis University
+* Aziza Mirsaidova, Microsoft
+* Jamshidbek Mirzakhalov, Monic AI
+* Saliha Muradoğlu, Australian National University
+* Fırat Öter, Middle East Technical University
+* Arzucan Özgür, Bogaziçi University
+* Adnan Öztürel, Google
+* Gözde Gül Şahin, Koç University
+* Francis Tyers, Indiana University
+* Jonathan Washington, Swarthmore College
 
 == Topics of Interest
 
 We welcome submissions on, but not limited to, the following topics:
 
-* Computational linguistics of Turkic languages
-* Evaluation of NLP methods for Turkic languages
-* Building Turkic language NLP systems
-* Metrics and measures for Turkic languages NLP
-* Learning from small or sparse data in Turkic languages
-* Datasets, benchmarks, and software for Turkic languages
+* Computational linguistics: models of all aspects of linguistics in Turkic languages (e.g., semantics, syntax, lexicon, morphology)
+* Systems: Case studies on the construction of NLP systems for Turkic languages
+* Evaluation: Understanding the applicability of current NLP methods in Turkic languages
+* Metrics: New metrics and measures for evaluating NLP systems suitable to Turkic languages
+* Learning from sparse data: Novel methods for learning from small or sparse data in Turkic languages
+* Resources: Datasets, benchmarks, and software libraries for NLP models in Turkic languages
 
-== Submissions Guidelines
+== Hackathon
```

Comment:
`Hack Together` as a tentative title please. Let's discuss details next week otherwise

Reply:
Wording using proposed title added in 28cdb2947cabc533987be7a3d91cb19c19f8d213